changelog/fragments/1732656422-add-actionable-error-message-for-enroll-command.yaml

```diff
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# Change summary; a 80ish characters long description of the change.
+summary: Elastic agent returns an actionable error message when a root user tries to enroll an unprivileged agent that has been unenrolled via the fleet ui
```

Comment:
`has been unenrolled` - really?

Reply:
This indeed is not the case, it is all enroll commands except for the one initiated by the install command. Updating right away